app/src/ui/dialog/dialog.tsx

```diff
@@ -10,6 +10,11 @@ import { createUniqueId, releaseUniqueId } from '../lib/id-pool'
  */
 const dismissGracePeriodMs = 250
 
+/** The time (in milliseconds) that we should wait after focusing before we
+ * re-enable click dismissal. Note that this is only used on Windows.
+ */
+const DisableClickDismissalDelay = 500
```

Comment:
This value was determined entirely by testing on my VM. Event dispatch can be slow :hurtrealbad: